stream-chat-android-ui-common/src/main/res/values/strings.xml

```diff
@@ -18,4 +18,5 @@
 
     <!--Message Date-->
     <string name="stream_date_yesterday">Yesterday</string>
+    <string name="stream_ui_attachment_display_error">Error. File can\'t be displayed</string>
```

Comment:
Nit: it looks like this is string related to *Message Date* somehow

Reply:
Today I'm a bit distracted...